docs/celeborn_ratis_shell.md

```diff
@@ -22,6 +22,8 @@ license: |
 Celeborn uses Ratis to implement the HA function of the master, Celeborn directly introduces ratis-shell package into the project
 then it's convenient for Celeborn Admin to operate the master ratis service.
 
+Since 0.6.0, the ratis [RESTful API](webapi.md) is supported, which is more convenient to operate the ratis service, see details in the swagger: `http://<CELEBORN_HOST>:<CELEBORN_PORT>/swagger/#/Ratis`.
```

Comment:
I think it is enough for the docs.

<img width="1317" alt="image" src="https://github.com/user-attachments/assets/db865860-407e-49f8-86f2-c8835ca49268">

Reply:
> I think it is enough for the docs.
> 
> <img alt="image" width="1317" src="https://private-user-images.githubusercontent.com/6757692/375567784-db865860-407e-49f8-86f2-c8835ca49268.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2MTM0NzMsIm5iZiI6MTcyODYxMzE3MywicGF0aCI6Ii82NzU3NjkyLzM3NTU2Nzc4NC1kYjg2NTg2MC00MDdlLTQ5ZjgtODZmMi1jODgzNWNhNDkyNjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTFUMDIxOTMzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWM2YjNjODBkNTJlN2YwMTk1ZGQ4ZDU4M2RjYzcwNjRkNjEwZGNhNDM4YzcxNzQ4MTA5YjRjMjc3OTcyMzU0ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.VbxxIMOBnAyIUfHgEaPgyoXTD_L0aoQsFbIoe__15as">

Sure